core/domain/skill_services.py

```diff
@@ -24,6 +24,7 @@
 from core.domain import role_services
 from core.domain import skill_domain
 from core.domain import user_services
+from core.domain import state_domain
```

Comment:
We prefer to keep imports sorted in alphabetical order to make it easier to find them in files with lots of imports. This is why the lint check is failing on Circle CI with this error:
```
ERROR: /home/circleci/oppia/core/domain/skill_services.py Imports are incorrectly sorted.
--- /home/circleci/oppia/core/domain/skill_services.py:before	2019-12-03 21:04:13.283627
+++ /home/circleci/oppia/core/domain/skill_services.py:after	2019-12-03 21:26:21.601638
@@ -23,8 +23,8 @@
 from core.domain import opportunity_services
 from core.domain import role_services
 from core.domain import skill_domain
+from core.domain import state_domain
 from core.domain import user_services
-from core.domain import state_domain
 from core.platform import models
 import feconf
 import python_utils


FAILED   Import order checks failed
```
You can catch failures like these locally by running the presubmit checks (details in the [wiki](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change)).

Reply:
Done!